blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -116,4 +118,4 @@ PipelinePage.childContextTypes = {
 const selectors = createSelector([pipelineSelector],
     (pipeline) => ({ pipeline }));
 
-export default connect(selectors, actions)(documentTitle(PipelinePage));
+export default translate(['jenkins.plugins.blueocean.dashboard.Messages'], { wait: true })(connect(selectors, actions)(documentTitle(PipelinePage)));
```

Comment:
It would be nice if we could find a cleaner way of doing this, would it not? Having to add this `translate(blah, blah, blah .....` at the end of all these modules seems a bit tedious.